src/main/java/org/mastodon/mamut/MamutBranchViewTrackScheme.java

```diff
@@ -152,7 +158,8 @@ protected MamutBranchViewTrackScheme(
 				viewGraph,
 				highlightModel,
 				navigateFocusModel,
-				( timepointModel == null ) ? this.timepointModel : timepointModel,
```

Comment:
Removing this line will actually reintroduce a bug. Please don't remove that line. You may read the commit message that introduced this line to see what I'm talking about.

Reply:
Thanks for the hint. I added this line again together with a comment, explaining, why it is needed.

Reply:
I failed to reproduce the bug that is described here (https://github.com/mastodon-sc/mastodon/pull/201) when this line is removed. However, if change the line as you suggest, this introduces a bug into the fading.
Perhaps, you could show me how to reproduce the bug if that line is removed?

Reply:
After checking again, I could added this line again.
There was another error, which I solved by adding a second timepoint model. Now, there are two timepoint models, one that is for highlighting the current row of spots (which is overridden in trackschemehierarchy) and one that is used for computing the fading.

Reply:
And now we finally don't even need the second timepoint model.